rasa/core/training/story_reader/__init__.py

```diff
@@ -0,0 +1,3 @@
+from rasa.core.training.story_reader.yaml_story_reader import YAMLStoryReader
+from rasa.core.training.story_reader.markdown_story_reader import MarkdownStoryReader
+from rasa.core.training.story_reader.story_reader import StoryReader
```

Comment:
why?

Reply:
Not sure, but now I don't see why I need that.